src/components/basecamp/CreateUpdateModal.tsx

```diff
@@ -0,0 +1,43 @@
+import { Dialog } from '@headlessui/react'
+import { XMarkIcon } from '@heroicons/react/24/outline'
+import { ReactElement, useRef } from 'react'
+
+import { Button, ButtonVariant } from '../ui/BaseButton'
+
+interface CreateUpdateModalProps {
+  isOpen: boolean
+  onClose?: () => void
+  contentContainer: ReactElement
+}
+
+/**
+ * Modal for updating and creating objects (areas, organizations, etc) in our datastores.
+ */
+export default function CreateUpdateModal ({
```

Comment:
We're deprecating `@headlessui/react` and moving to `radix-ui`

Can you wrap the edit screen with a MobileDialog?  You can see an example here 

https://github.com/OpenBeta/open-tacos/blob/develop/src/components/edit/Triggers.tsx#L95

---